### PR TITLE
Other user/full week

### PR DESCRIPTION
**Pull request by shanth5100**

### CODE REVIEW

File: androidapp/utils/src/main/res/values-es/strings.xml [hunk 1]
```diff
@@ -1,4 +1,19 @@
...
+    <string name="sunday">Dom</string>
+
+    <!-- Full week day names -->
+    <string name="monday_full">lunes</string>
+    <string name="tuesday_full">martes</string>
+    <string name="wednesday_full">miércoles</string>
+    <string name="thursday_full">jueves</string>
+    <string name="friday_full">viernes</string>
+    <string name="saturday_full">sábado</string>
+    <string name="sunday_full">domingo</string>
```
Comment (vj12354) [inline androidapp/utils/src/main/res/values-es/strings.xml]: Any chance if these can be capital.?

File: androidapp/utils/src/main/res/values-fr/strings.xml [hunk 2]
```diff
@@ -1,4 +1,19 @@
...
+    <string name="sunday">dim</string>
+
+    <!-- Full week day names -->
+    <string name="monday_full">lundi</string>
+    <string name="tuesday_full">mardi</string>
+    <string name="wednesday_full">mercredi</string>
+    <string name="thursday_full">jeudi</string>
+    <string name="friday_full">vendredi</string>
+    <string name="saturday_full">samedi</string>
+    <string name="sunday_full">dimanche</string>
```
Comment (vj12354) [inline androidapp/utils/src/main/res/values-fr/strings.xml]: if possible starting with Capital letter. 
Example: Lundi, Mardi ?

**Review by vj12354 — COMMENTED** [review]
Can you please check if starting capital letter can be provided, where ever possible.?

File: androidapp/utils/src/main/res/values-es/strings.xml (hunk 1 above)
Comment (shanth5100) [reply androidapp/utils/src/main/res/values-es/strings.xml]: > Any chance if these can be capital.?

Ok Sir, I will change it.

**Review by vj12354 — APPROVED** [review]
All looking good with de, es, fr, hi, ru. Merging now.

**Pull request merged**